src/ports/postgres/modules/utilities/control.py_in

```diff
@@ -24,56 +25,66 @@ HAS_FUNCTION_PROPERTIES = m4_ifdef(<!__HAS_FUNCTION_PROPERTIES__!>, <!True!>, <!
 UDF_ON_SEGMENT_NOT_ALLOWED = m4_ifdef(<!__UDF_ON_SEGMENT_NOT_ALLOWED__!>, <!True!>, <!False!>)
 
 
-class EnableOptimizer(object):
+class OptimizerControl(object):
 
     """
     @brief: A wrapper that enables/disables the optimizer and
         then sets it back to the original value on exit
```

Comment:
You might want to consider updating the description to include reference to the guc `optimizer_control` and its usage.